app/models/vm/operations.rb

```diff
@@ -14,7 +14,14 @@ module Vm::Operations
   end
 
   def cockpit_url
-    URI::HTTP.build(:host => ipaddresses.first, :port => 9090).to_s
+    URI::HTTP.build(:host => get_ipv4_address, :port => 9090).to_s
```

Comment:
What if the environment has ipv6-only connectivity? (yes, that's actually thing).

Perhaps we can have a hearustic based on what `MiqServer.my_server.ipaddress` has. Or what the webclient connection has.

Or alternatively we should just drop link-local from the list?